integration/test_magic_folder.py

```diff
@@ -193,19 +283,51 @@ def test_bob_conflicts_with_alice_preexisting(magic_folder):
 
```

Comment:
is everything from this point on the same as `test_bob_conflicts_with_alice_fresh`?  If so, can this test just call `test_bob_conflicts_with_alice_fresh` instead of duplicating all of the implementation?